component/Lists/list.html

```diff
@@ -0,0 +1,117 @@
+            </div>
+            <div class="flex flex-column items-start wt-100 justify-even m-y-5">
+                <div class=" avatar-head ">
+                    <span class="text-xm ">Spaced Lists</span>
+                </div>
+                <div class=" box-shadow-2 m-auto wt-50">
+                    <ul class="flex justify-around ">
+                        <li class="style-none pd-3 ">
+                            List 1
+                        </li>
+                        <li class="style-none pd-3 ">
+                            List 2
+                        </li>
+                        <li class="style-none pd-3 ">
+                            List 3
+                        </li>
+                        <li class="style-none pd-3">
+                            List 4
+                        </li>
+                        <li class="style-none pd-3">
+                            List 5
+                        </li>
+                    </ul>
+                </div>
+                <div class=" m-y-5 m-auto">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=16px&ph=25px&ln=false&fl=1&fm=Hack&fs=11px&lh=133%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522flex%2520justify-around%2520%2522%253E%250A%2520%2520%2520%253Cli%2520class%253D%2522style-none%2520pd-3%2520%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520List%25201%250A%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%253Cli%2520class%253D%2522style-none%2520pd-3%2520%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520List%25202%250A%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%253Cli%2520class%253D%2522style-none%2520pd-3%2520%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520List%25203%250A%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%253Cli%2520class%253D%2522style-none%2520pd-3%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520List%25204%250A%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%253Cli%2520class%253D%2522style-none%2520pd-3%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520List%25205%250A%2520%2520%2520%253C%252Fli%253E%250A%253C%252Ful%253E"
+                        style="width: 35rem; height: 15rem; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <div class=" avatar-head m-y-5">
+                    <span class="text-xm ">Stacked Lists</span>
+                </div>
+                <div class=" box-shadow-2 m-auto wt-50 ">
+                    <ul class="flex flex-column ">
+                        <li class="style-none pd-3 ">
+                            List 1
+                        </li>
+                        <li class="style-none pd-3 ">
```

Comment:
Suggestion: I guess it should have an explicit list class in your list.css. You are using common class to style the list.